framework/src/modules/chain/chain.js

```diff
@@ -505,9 +509,19 @@ module.exports = class Chain {
 			this.channel.publish('chain:blocks:change', block);
 		});
 
+		this.transactionPool.on(EVENT_UNCONFIRMED_TRANSACTION, transaction => {
+			this.logger.debug(`Broadcasting transaction ${transaction.id}`);
```

Comment:
Let's make the message more clear.

Reply:
Maybe we could use Bunyan's metadata handling such as:
`this.logger.debug({transactionId: 'theTrsId'}, 'Broadcasting transaction'}`
This will print the logs in a parseable JSON-structured format

Reply:
Done